.github/workflows/release.yml

```
@@ -0,0 +1,12 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v4
```

This action is deprecated since last month, we should try out the new one

```suggestion
      - uses: googleapis/release-please-action@v4
```

---

[Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/41/commits/0262f3fa8ab3c0367a8a4ca97d1bc719ec894307#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R10)